### PR TITLE
Add license terms to README

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/pyvec/naucse.python.cz/issues/88

<del>Can only be merged after all the authors agree.</del> Agreed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,17 @@
...
 Brzy dostupné na naucse.python.cz.
+
+
+## License
+
+The code is licensed under the terms of the MIT license, see LICNESE.MIT file
+for full text. By contributing code to this repository, you agree to have it
+licensed with the same license.
+
+Content has it's own license specified in the appropriate matadata.
```
Comment (encukou) [inline README.md]: typo: `its`

File: README.md [hunk 2]
```diff
@@ -3,3 +3,17 @@
...
+Only [free licenses] are used. By contributing to an already licensed content,
```
Comment (encukou) [inline README.md]: Please use the term, `free content licenses`.

At the end of the line, change `content` to `document`, or get rid of the `an`.

File: README.md [hunk 3]
```diff
@@ -3,3 +3,17 @@
...
+you agree to have it licensed with the same license. When contributing new
```
Comment (encukou) [inline README.md]: `licensed under`

Do we want to make it clear that you can add yourself to the list of authors when making a contribution?

**Comment by hroncok** [comment]
Amended.

**Pull request merged**